### PR TITLE
[palace] Add version and explicit BLAS support

### PR DESCRIPTION
**Pull request by stephenmsachs**
BLAS and LAPACK is not automatically found in case of using armpl.

### CODE REVIEW

File: var/spack/repos/builtin/packages/palace/package.py [hunk 1]
```diff
@@ -114,6 +115,13 @@ def cmake_args(self):
         if "+mumps" in self.spec:
             args += ["-DMUMPS_REQUIRED_PACKAGES=MPI;MPI_Fortran"]
 
+        # BLAS support
```
Comment (sebastiangrimberg) [inline var/spack/repos/builtin/packages/palace/package.py]: Suggested cleaner comment:

```
        # BLAS/LAPACK linkage
        args += [
            "-DBLAS_LIBRARIES={0}".format(self.spec["blas"].libs.joined(";")),
            "-DLAPACK_LIBRARIES={0}".format(self.spec["lapack"].libs.joined(";")),
        ]
```

**Review by sebastiangrimberg — COMMENTED** [review]
Looks good, just one minor style suggestion before merging

File: var/spack/repos/builtin/packages/palace/package.py [hunk 2]
```diff
@@ -115,7 +115,7 @@ def cmake_args(self):
         if "+mumps" in self.spec:
             args += ["-DMUMPS_REQUIRED_PACKAGES=MPI;MPI_Fortran"]
 
-        # BLAS support
+        # BLAS/LAPACK linkage
         args += [
             "-DBLAS_LIBRARIES={0}".format(self.spec["blas"].libs.joined(";")),
             # LAPACK support
```
Comment (sebastiangrimberg) [inline var/spack/repos/builtin/packages/palace/package.py]: Sorry, I intended that this line can be deleted with the updated comment.

Comment (stephenmsachs) [reply var/spack/repos/builtin/packages/palace/package.py]: Got you. I didn't see that other comment.

**Review by sebastiangrimberg — APPROVED** [review]

**Review by alalazo — APPROVED** [review]

**Pull request merged**